### DISCUSSION TITLE
Error when trying to run the example code

### DISCUSSION THREAD

**Discussion by mendo10jm** [golioth/golioth-esp-idf-sdk#68] [Q&A] [answered]
Hi! I am trying to run the last version of the golioth-esp-idf-sdk in my esp32 with esp-idf 4.4.1 version and I always get the following error ->
E (5654) libcoap: libcoap not compiled for DTLS with Mbed TLS - update Mbed TLS to include DTLS
I (5684) golioth_coap_client: Start CoAP session with host: coaps://coap.golioth.io
E (5694) libcoap: coap_new_client_session*: DTLS not supported
E (5694) golioth_coap_client: coap_new_client_session() failed

How can I solve this?

Tank you so much!

**Comment by ncmiller** [comment, marked as answer]
Hi @mendo10jm - which example are you trying to run? Is it `golioth_basics`?

The only way you would see this error is if `CONFIG_MBEDTLS_SSL_PROTO_DTLS` is not set (or set to `n`) in `sdkconfig`. Check your `sdkconfig` file to see what that is set to. It should be set to `y`. 

All of the examples set that config value to `y` by default via `sdkconfig.defaults` (e.g. https://github.com/golioth/golioth-esp-idf-sdk/blob/main/examples/golioth_basics/sdkconfig.defaults#L1).

**Reply by mendo10jm** [reply]
Thank you @ncmiller I was using wrong sdkconfig! Every thing works now!
